host_guest/Isaacs_clip/host_guest/README.md

```diff
@@ -1,9 +1,9 @@
 # Host
 
-- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb).
+- Provided is the Trimertrip host for this challenge in PDB, Tripos MOL2, and SDF file format (trimertrip.mol2, trimertrip.sdf, trimertrip.pdb). The structure was provided by Lyle Isaacs, from a crystal structure. The crystal structure will not be released. The original starting structure will be added here once the challenge ends
 - Trimertrip host files were created as follows:
   - Original starting structure (in `CIF` file format, which will be added here once the challenge ends) was converted to a PDB using OpenEye toolkits. Water, a guest, a single Na+ ion, and connectivity records were manually removed. OpenEye toolkit was used to normalize the molecule, add explicit hydrogen, assign partial charges using AM1-BCC and save the resulting molecule to a `MOL2` file. The SYBYL atom types in the `MOL2` file were switched to GAFF2 atom types using antechamber, as the parmchk2 toolkit used afterwards expects GAFF/GAFF2 atom types. The parmchk2 toolkit was used to generate parameters and a `frcmod` file and these files were used in tleap (adding TIP3P water and sodium chloride) to generate amber files. The OpenMM toolkit was used to briefly run some dynamics to generate an amber compatible NetCDF coordinate file with initial velocities. Afterwards, an annealing procedure was used where the structure was heated from 0K to 450K for 30ps, then kept at kept at the high temperature for 5ns, then cooled to room temperature (273K) for 30ps and then kept at this temperature for 10ns. The RMSD of the final Trimertrip structure from the starting crystal structure is 3 Angstroms.
-  - File creation: The `pytraj` toolkit was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2` file was then used to generate a SDF and PDB file.
+  - File creation: The `pytraj` toolkit (the python interface of `cpptraj`, which is apart of `AmberTools`) was used to remove the solvent (water, Na+ and Cl-) and create a `MOL2` file, then the `antechamber` toolkit was used to convert GAFF2 atom types to SYBYL/Tripos atom type, the resulting `MOL2` file was read into OpenEye and written out to a new standards-compliant `MOL2` file, this `MOL2` file was then used to generate a SDF and PDB file using OpenEye tools.
```

Comment:
"a part", not "apart". Those two mean almost opposite things, oddly.

Reply:
Changed.